pkg/actuators/machine/actuator.go

```diff
@@ -352,16 +352,24 @@ func (a *Actuator) Update(context context.Context, cluster *clusterv1.Cluster, m
 		return errMsg
 	}
 
-	instances, err := getRunningInstances(machine, client)
+	// Get just running instances
+	runningInstances, err := getRunningInstances(machine, client)
 	if err != nil {
 		glog.Errorf("error getting running instances: %v", err)
 		return err
 	}
-	glog.Infof("found %d instances for machine", len(instances))
+	// Get all instances not terminated.
+	existingInstances, err := getExistingInstances(machine, client)
+	if err != nil {
+		glog.Errorf("error getting existing instances: %v", err)
+		return err
+	}
+	existingLen := len(existingInstances)
```

Comment:
we can do this inline and avoid an extra variable

Reply:
We use existingLen more than once, though it may be O(1) under the hood, not sure.

Reply:
> we can do this inline and avoid an extra variable

I would be in favour of dropping the temporary too.